java/src/jmri/jmrix/openlcb/swing/monitor/MonitorPane.java

```diff
@@ -293,7 +293,8 @@ void format(String prefix, boolean extended, int header, int len, int[] content)
             }
         } else {
             // control type
-            String alias = "0x" + Integer.toHexString(header & 0xFFF).toUpperCase();
+            var aliasAsHex = "000"+Integer.toHexString(header & 0xFFF).toUpperCase();
+            String alias = "0x" + (aliasAsHex).substring(aliasAsHex.length()-3,aliasAsHex.length());
```

Comment:
I suggest that you replace these two lines with:
`String alias = String.format("0x%03X", header & 0xFFF);`

0x = Prepend the value with 0x
03 = Three digits padded with 0
X = uppercase letters (x = lowercase)

Reply:
Thanks!  Both changes made.